1-getting-started-lessons/2-github-basics/README.md

```diff
@@ -134,14 +134,16 @@ Let's say you have a folder locally with some code project and you want to start
    This creates a _remote_, or connection, named "origin" pointing at the GitHub repository you created earlier.
 
 1. **Send local files to GitHub**. So far you've created a _connection_ between the local repo and the GitHub repo. Let's send these files to GitHub with the following command `git push`, like so: 
+   
+   > Note, replace ```main``` with the branch name you want to push to GitHub.

```

Comment:
We want to encourage folks to use 'main', so let's say:

"Note, your branch name may be different by default from 'main'"

Reply:
Ok, i will update it. Thanks